### PR TITLE
Tidy how section issue #16

### PR DESCRIPTION
**Pull request by nelsonic**
+ [x] Tidy up "How?" section a bit and add links to relevant issues. for #16

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -208,7 +208,7 @@ or curious about UI/Design in general,
 you _might_ find certain features familiar.
 You may think:
 "_note-taking similar to Evernote_",
-"_collaborative editing like Google Docs_"
+"_collaborative editing like Google Docs_"x§
```
Comment (iteles) [inline README.md]:
```suggestion
"_collaborative editing like Google Docs_"
```

Comment (iteles) [reply README.md]: Assuming this was an unintentional add

**Review by iteles — APPROVED** [review]
@nelsonic Thanks for the update ❤️

File: README.md (hunk 1 above)
Comment (nelsonic) [reply README.md]: yeah, must have been [Nala](https://github.com/nelsonic/nelsonic.github.io/issues/379) walking on my keyboard when I was AFK ... 🐈 😉

**Pull request merged**